src/sbvr-api/hooks.ts

```diff
@@ -50,24 +50,24 @@ export class SideEffectHook extends Hook {
 }
 
 // The execution order of rollback actions is unspecified
-export const rollbackRequestHooks = Promise.method(
-	<T extends InstantiatedHooks<any>>(hooks: T | undefined): void => {
-		if (hooks == null) {
-			return;
-		}
-		settleMapSeries(
-			_(hooks)
-				.flatMap()
-				.compact()
-				.value(),
-			hook => {
-				if (hook instanceof SideEffectHook) {
-					return hook.rollback();
-				}
-			},
-		);
-	},
-);
+export const rollbackRequestHooks = <T extends InstantiatedHooks<any>>(
+	hooks: T | undefined,
+): void => {
+	if (hooks == null) {
+		return;
```

Comment:
In another PR, you had explained that hook functions don't need to return promises... I guess it applies here as well, because `Promise.method` was removed.

Reply:
This is actually a different situation, but here we are always returning void and never anything to actually wait on (and we never tried to wait on it either) so the `Promise.method` was unnecessary/confusing and would also have been a slight performance hit, probably neglible but still better to avoid